.ci/JenkinsApiCompatibility

```diff
@@ -1,5 +1,5 @@
 // Check corda-api compatibility with downstream consumers which implement CordApps
-@Library('corda-shared-build-pipeline-steps@5.1') _
+@Library('corda-shared-build-pipeline-steps@jacob/ES-1398') _

```

Comment:
I think we'll have to wait for this pipeline change to be merged into 5.1 (in corda-shared-build-pipeline-steps) and not change this here

Reply:
Thanks, once the pipeline PR is merged I'll change this and mark as ready for review